control/check_install.sh

```diff
@@ -45,30 +46,24 @@ fi
 
 if [ ! -f "${FAB_CONFIG}" ]
 then
-    for OLD_FAB_DIR in "${COMMCARE_CLOUD_REPO}/../commcare-hq/deployment/commcare-hq-deploy" \
-                       "${COMMCARE_CLOUD_REPO}/../commcare-hq-deploy" \
-                       "${COMMCARE_CLOUD_REPO}/fab"
-    do
-        if [ -d "${OLD_FAB_DIR}" ]
+    OLD_FAB_CONFIG_DIR="${COMMCARE_CLOUD_REPO}/src/commcare_cloud/fab"
+    if [ -d "${OLD_FAB_CONFIG_DIR}" ]
+    then
+        OLD_FAB_CONFIG="${OLD_FAB_CONFIG_DIR}/config.py"
+        if [ -f "${OLD_FAB_CONFIG}" ]
```

Comment:
@skodde said (offline):
> since you removed the for loop, you could also remove having the check separately on the dir and the config file, and keep only the latter

Reply:
Updated :)